iina/Base.lproj/Localizable.strings

```diff
@@ -80,6 +80,8 @@
 "osd.file_error" = "Error reading file";
 "osd.cannot_login" = "Cannot login";
 "osd.canceled" = "Canceled";
+"osd.cannotConnect" = "Cannot connect";
```

Comment:
`cannot_connect`

Reply:
Fixed in new commit.
Thanks for catching!